cms/static/sass/_base.scss

```diff
@@ -84,7 +84,7 @@ h1 {
     position: relative;
     top: ($baseline/4);
     display: block;
-    color: $gray-l2;
+    color: $gray-d2;
```

Comment:
It's this:

<img width="979" alt="screen shot 2016-07-08 at 12 25 51 pm" src="https://cloud.githubusercontent.com/assets/2112024/16694250/261747f6-4507-11e6-9c52-5305a2439d67.png">

And it was previously too light.